modules/@apostrophecms/ui/ui/apos/scss/_admin.scss

```diff
@@ -0,0 +1,15 @@
+$a-max-canvas: 1168px;
+
+.apos-editor-context {
+  display: flex;
+  padding-top: 50px;
+  background-color: var(--a-base-9);
+}
+
+.apos-editor-canvas {
```

Comment:
Let's also set left and right padding. When the new area UI is done it'll need to be floating to the left of the canvas, so we'll need to always have room for it.

Reply:
Trying my best to avoid messing with the inner canvas, as that is the real representation of the site.

Instead I flex centered the canvas and set margins to pad the context container. The margin value could change as this progresses

Reply:
I don't understand. How is the area UI supposed to float on the left of the canvas like this?

![Screen Shot 2020-09-09 at 10 46 23 AM](https://user-images.githubusercontent.com/1155984/92621677-b76c7080-f289-11ea-9155-53f86f4b2f7d.png)

Reply:
control positioning happening in [3/widget-interactions](https://github.com/apostrophecms/apostrophe/tree/3/widget-interactions)

Reply:
🆗